executable/Server.re

```diff
@@ -185,20 +50,23 @@ let user =
             p.name
           ),
           field(
-            "role", ~args=Arg.[], ~typ=non_null(role), ~resolve=(_info, p) =>
+            "role",
+            ~args=Arg.[],
+            ~typ=non_null(roleResolver),
+            ~resolve=(_info, p) =>
             p.role
           ),
           field(
             "summary",
             ~args=Arg.[],
-            ~typ=non_null(summary),
+            ~typ=non_null(Summary.resolver),
             ~resolve=(_info, _p) =>
-            mockedSummary
+            {unread: 0, total: 0, projects: 0, organizations: 0}
```

Comment:
I would leave the mocked for now as this is indeed not hooked up but ok

Reply:
this is the mocked, just removed the constant since now is everything on the same file